Jenkinsfile

```diff
@@ -24,6 +24,7 @@ stage("Unit Test") {
           pip uninstall -y autogluon.mxnet
           pip uninstall -y autogluon.extra
           pip uninstall -y autogluon.tabular
+          pip uninstall -y autogluon.features
```

Comment:
I think that it should be in `autogluon.core` should we install another `autogluon.features` module?

Reply:
What do you mean? Are you saying that autogluon.core should depend on autogluon.features?

Reply:
Per-offline discussion, @Innixma mentioned that we may later expand the functionality of `autogluon.features` so it's beneficial to have a separate `autogluno.features` package.